From 24dd10608bfb554390c17f709a5afa30060c994b Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Wed, 31 Oct 2018 10:49:16 +0100
Subject: [PATCH] Add OPENSSL_INCLUDE_DIR

Add OPENSSL_INCLUDE_DIR to be able to override openssl include directory
as -I/usr/include/openssl can't be used when cross-compiling

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: https://github.com/paulusmack/ppp/pull/107]
---
 pppd/Makefile.linux | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pppd/Makefile.linux b/pppd/Makefile.linux
index 8d5ce99..b258d86 100644
--- a/pppd/Makefile.linux
+++ b/pppd/Makefile.linux
@@ -84,6 +84,7 @@ USE_LIBUTIL=y
 MAXOCTETS=y
 
 INCLUDE_DIRS= -I../include
+OPENSSL_INCLUDE_DIR= /usr/include/openssl
 
 COMPILE_FLAGS= -DHAVE_PATHS_H -DIPX_CHANGE -DHAVE_MMAP
 
@@ -137,7 +138,7 @@ endif
 
 ifdef NEEDDES
 ifndef USE_CRYPT
-CFLAGS   += -I/usr/include/openssl
+CFLAGS   += -I$(OPENSSL_INCLUDE_DIR)
 LIBS     += -lcrypto
 else
 CFLAGS   += -DUSE_CRYPT=1
-- 
2.17.1

